data/service/raw-data-service.js

```diff
@@ -808,6 +805,12 @@ exports.RawDataService = DataService.specialize(/** @lends RawDataService.protot
      *                             [addRawData()]{@link RawDataService#addRawData}
      *                             call that invoked this method.
      */
+     mapRawDataToObject: {
```

Comment:
doesn't that break backward compatibility or are we now in position to do so?

Reply:
will fix - the code comments were not lined up so I put it over the right function.  There was a merge conflict and I removed the wrong block.